s2wav/tiling.py

```diff
@@ -0,0 +1,63 @@
+"""Script defining basic mathematical functions needed to carry out the tiling of the harmonic space. 
+"""
+import numpy as np
+import pytest
+
+
+def tiling_integrand(t: float, lam: float) -> float:
```

Comment:
Much better name but we may want to be slightly more specific so avoid confusion later down the line. How about `tiling_integrand_s2dw()`?

Reply:
Since I think we discussed s2dw refers to a prior Fortran code, maybe we should come up with another label to signify the wavelets case? Not sure what would be easily understood @CosmoMatt @jasonmcewen

Reply:
Good point @alicjapolanska, perhaps we just keep this for now and when we come to add other functions we can rename these. Should be good to pull this into main, just need @jasonmcewen to approve changes etc.